packages/flutter_tools/BUILD.gn

```diff
@@ -75,3 +77,26 @@ dart_tool("fuchsia_tester") {
     "$flutter_root/shell",
   ]
 }
+
+sdk_atom("fuchsia_asset_builder_sdk") {
+  domain = "dart"
```

Comment:
It would be easier to put this in the `exe` domain.

Reply:
Done.